src/pyfmi/fmi3.pyx

```diff
@@ -794,7 +803,7 @@ cdef class FMUModelBase3(FMI_BASE.ModelBase):
             data_causality   = FMIL3.fmi3_import_get_variable_causality(variable)
             data_initial     = FMIL3.fmi3_import_get_variable_initial(variable)
             data_type        = FMIL3.fmi3_import_get_variable_base_type(variable)
-            #If only variables with start are wanted, check if the variable has start
+            # If only variables with start are wanted, check if the variable has start
 
             # TODO: Discuss if we want to support also regular integers as inputs
```

Comment:
By the way we can remove this comment "# TODO: ...", I realize I forgot to remove it in the PR where I made the Enums into IntEnum.